modules/core/src/main/java/org/apache/synapse/config/xml/MessageStoreMediatorFactory.java

```diff
@@ -70,4 +85,16 @@ protected Mediator createSpecificMediator(OMElement elem, Properties properties)
     public QName getTagQName() {
        return STORE_Q;
     }
+
+    public boolean checkForBraces(OMAttribute atr){
+        if((atr.getAttributeValue().startsWith("{")) && (atr.getAttributeValue().endsWith("}")))
```

Comment:
Can "atr" be null?

Reply:
No it can't be. The method is called after null-checking the atr.